README.md

```diff
@@ -1,50 +1,26 @@
-# PDS Template Repository for Python
+# Registry Sweepers
 
-This is the template repository for PDS's Python projects.
+This module enable to sweep the PDS Registries by:
 
-This repository aims at being a base for new python repositories used in PDS. It guides developers to ease the initialization of a project and recommends preferred options to standardize developments and ease maintenance. Simply click the <kbd>Use this template</kbd> button ↑ (or use [this hyperlink](https://github.com/NASA-PDS/pds-template-repo-python/generate)).
+- adding the superseded_by property on the non latest products
+- adding ancestry to the products
 
 
-## 🏃 Getting Started With This Template
-
-See our wiki page for more info on setting up your new repo. You can remove this section once you have completed the necessary start-up steps.
-
-https://github.com/NASA-PDS/nasa-pds.github.io/wiki/Git-and-Github-Guide#creating-a-new-repo
-
-**👉 Important!** You must assign the teams as mentioned on the wiki page above! At a minimum, these are:
-
-| Team                                | Permission |
-| ----------------------------------- | ---------- |
-| `@NASA-PDS/pds-software-committers` | `write`    |
-| `@NASA-PDS/pds-software-pmc`        | `admin`    |
-| `@NASA-PDS/pds-operations`          | `admin`    |
-
----
-
-# My Project
-
-This is the XYZ that does this, that, and the other thing for the Planetary Data System.
-
-Please visit our website at: https://nasa-pds.github.io/pds-my-project
-
-It has useful information for developers and end-users.
-
 ## Prerequisites
 
-Include any system-wide requirements (`brew install`, `apt-get install`, `yum install`, …) **Python 3** should be used regardless as [Python 2 reached end-of-life on January 1st, 2020](https://pythonclock.org/).
-
+python 3.9
 
 ## User Quickstart
 
 Install with:
 
-    pip install my_pds_module
-
-If possible, make it so that your program works out of the box without any additional configuration—but see the [Configuration](###configuration) section for details.
+    pip install registry-sweeper
 
 To execute, run:
 
-    (put your run commands here)
+    python src/pds/registrysweepers/provenance.py --help
```

Comment:
If you're doing a `pip install`, then you won't have access to `src/pds/…`. Instead the "To execute, run:" instructions should instead give the name of the installed console script.

Reply:
The current setup is that [the pip-installation of] registry-sweepers provides a library, which will be [called in a context specific to its deployment](https://github.com/NASA-PDS/registry-sweepers/blob/main/docker/sweepers_driver.py#L138-L139).

So I think this content is more likely to mislead than elucidate.

@tloubrieu-jpl if it's critical that we have readme content asap, do you want me to take a crack at it later?

Reply:
@alexdunnjpl can you make a quick update here for this?

Reply:
@jordanpadams done